### PR TITLE
support zoom with numpad keys

### PR DESCRIPTION
**Pull request by kevinushey**
### Intent

Addresses https://github.com/rstudio/rstudio/issues/12242.

### Approach

Add keybindings for the NumpadAdd and NumpadSubtract key codes, and use it.

### Automated Tests

N/A

### QA Notes

Test via notes in https://github.com/rstudio/rstudio/issues/12242.

### Documentation

N/A

### Checklist

- [x] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [x] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [x] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [x] This PR passes all local unit tests

### CODE REVIEW

**Review by gtritchie — COMMENTED** [review]
I built it on Windows, and it doesn't work. Ctrl+Numpad+/- is a no-op. I tried with NumLock on and off.

On Mac, it works.

Curiously, without your change (e.g. the latest daily build) Cmd+Numpad- also works, but Cmd+Numpad+ is a no-op (on Mac).

The number of people on Mac using a full keyboard is probably 0.02% of those doing so on Windows, so not sure this is worth the code churn it if it doesn't work on Windows?

File: src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml [hunk 1]
```diff
@@ -741,6 +741,8 @@ See `/src/gwt/tools/i18n-helpers/README.md`.
       <shortcutgroup name="View">
          <shortcut refid="zoomIn" value="Meta+=" if="org.rstudio.core.client.BrowseCap.isMacintoshDesktop()"/>
          <shortcut refid="zoomOut" value="Meta+-" if="org.rstudio.core.client.BrowseCap.isMacintoshDesktop()"/>
+         <shortcut refid="zoomIn" value="Meta+NumpadAdd"/>
+         <shortcut refid="zoomOut" value="Meta+NumpadSubtract"/>
```
Comment (kevinushey) [inline src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml:745]:
```suggestion
         <shortcut refid="zoomIn" value="Cmd+NumpadAdd"/>
         <shortcut refid="zoomOut" value="Cmd+NumpadSubtract"/>
```

**Review by gtritchie — APPROVED** [review]
Working now on Mac and Windows. Didn't try Linux.

You need to include the updated Commands.cmd.xml.MD5.

**Comment by kevinushey** [comment]
Thanks!

**Pull request merged**